### PR TITLE
fix: tooltip behaviour

### PR DESCRIPTION
**Pull request by matthieucrouzet-sym**

### CODE REVIEW

File: src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -60,8 +61,25 @@ export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'as'
   type?: 'hint' | 'tooltip';
   /** if true, the tooltip should be displayed */
   visible?: boolean;
+  debounceTime?: number;
```
Comment (symphony-adnane) [inline src/components/tooltip/Tooltip.tsx:64]: The name imho is very "implementation linked", we talk more about a hoverTimeout here I think
Also a comment for storybook is welcome :)

**Review by symphony-adnane — COMMENTED** [review]
Just a small naming change suggestion
Implementation works for me

File: src/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -120,13 +148,17 @@ const Tooltip: React.FC<TooltipProps> = ({
             id={id}
             role="tooltip"
             ref={setPopperElement}
-            className={ type === 'tooltip' ? 'tk-tooltip' : 'tk-hint' }
+            className={`${type === 'tooltip' ? 'tk-tooltip' : 'tk-hint'} ${
```
Comment (symphony-jean-michael) [inline src/components/tooltip/Tooltip.tsx:151]: You can use the 'classname' lib ;) (https://github.com/JedWatson/classnames#readme)
```
import classnames from 'classnames';
...
className={`classname('tk-tooltip': tooltip, 'tk-hint': !tooltip, className)} 
```

**Review by symphony-jean-michael — COMMENTED** [review]
Don't hesitate to add a description to your PR. It helps us to better understand the PR and provide a better code review. 😉 
( With a screenshot or even better a video/gif 😃 )

**Review by symphony-jean-michael — APPROVED** [review]

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**